### PR TITLE
Feign compression documentation

### PR DESCRIPTION
**Pull request by jmnarloch**
This is a fallow up for the pull request: https://github.com/spring-cloud/spring-cloud-netflix/pull/490 that adds the docs for this feature.

### CODE REVIEW

File: docs/src/main/asciidoc/spring-cloud-netflix.adoc [hunk 1]
```diff
@@ -126,7 +126,7 @@ application Spring Boot Actuator current health check status.  Which means
 that after successful registration Eureka will always announce that the
 application is in 'UP' state. This behaviour can be altered by enabling
 Eureka health checks, which results in propagating application status
-to Eureka, as a consequence every other application will not sending
+to Eureka> As a consequence every other application won't be sending
```
Comment (spencergibb) [inline docs/src/main/asciidoc/spring-cloud-netflix.adoc]: Looks funny. Period?

Comment (jmnarloch) [inline docs/src/main/asciidoc/spring-cloud-netflix.adoc]: A, yes.

Do I need rabbitmq to run the tests now?

**Comment by spencergibb** [comment]
Merged with some grammar fixes in 82b21f34f59a90fc3293d9d2b474ea1a97b5fc57.

Thanks for the follow up docs @jmnarloch!

**Comment by spencergibb** [comment]
@jmnarloch rabbit has been required since January (at least that's when we added it to travis).

**Pull request closed without merging**